src/machines/java/com/enderio/machines/common/lang/MachineLang.java

```diff
@@ -37,6 +37,10 @@ public class MachineLang {
     public static final Component TOO_MANY_SPAWNER = REGISTRATE.addLang("gui", EnderIO.loc("spawner.too_many_spawner"), "Too many spawners");
     public static final Component UNKNOWN = REGISTRATE.addLang("gui", EnderIO.loc("spawner.unknown"), "Unknown mob");
     public static final Component OTHER_MOD = REGISTRATE.addLang("gui", EnderIO.loc("spawner.other_mod"), "Other mod");
+    public static final Component PHOTOVOLTAIC_CELL = REGISTRATE.addLang("tooltip", EnderIO.loc("photovoltaic_cell/main"), "Solar Power!");
+    public static final Component PHOTOVOLTAIC_CELL_ADVANCED = REGISTRATE.addLang("tooltip", EnderIO.loc("photovoltaic_cell/advanced"), "Produces Power during daylight hours");
```

Comment:
Can we maybe give these slightly more descriptive field names than 1,2 etc.? just for ease so we don't have to double check lang keys :) Tbf thats more of a thing that needs done overall, feel free to just create an issue if you want to defer it to later.

Reply:
I wouldn't do that, as they are ordered and only make sense in order

Reply:
Thats fair :)

Reply:
Thats fair :)